### ISSUE TITLE
SKYTHRURMS on 20240821, tiles 27156, 27117, expid 249649-249651

### ISSUE THREAD

**Issue by schlafly** [desihub/desisurveyops#207]
QA reports large SKYTHRURMS on 20240821, tiles 27156, 27117, expid 249649 - 249651.  I don't actually see anything wrong with the data and can't think of a case where SKYTHRURMS has told us something useful before, so I don't really know what to do with this.  249650 & 249651 are particularly interesting because almost all R cameras have high SKYTHRURMS but the BZ cameras do not, and we expect TPCORR to be fairly achromatic.  These are bright tiles with pretty fine looking redshifts, so I'm inclined to ignore them, but @julienguy expressed some interest in these tiles.

**Comment by daqiii** [comment]
What are SKYTHRUMS?
Klaus

From: Eddie Schlafly ***@***.***>
Date: Tuesday, August 27, 2024 at 1:16 PM
To: desihub/desisurveyops ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [desihub/desisurveyops] SKYTHRURMS on 20240821, tiles 27156, 27117, expid 249649-249651 (Issue #207)
QA reports large SKYTHRURMS on 20240821, tiles 27156, 27117, expid 249649 - 249651. I don't actually see anything wrong with the data and can't think of a case where SKYTHRURMS has told us something useful before, so I don't really know what


QA reports large SKYTHRURMS on 20240821, tiles 27156, 27117, expid 249649 - 249651. I don't actually see anything wrong with the data and can't think of a case where SKYTHRURMS has told us something useful before, so I don't really know what to do with this. 249650 & 249651 are particularly interesting because almost all R cameras have high SKYTHRURMS but the BZ cameras do not, and we expect TPCORR to be fairly achromatic. These are bright tiles with pretty fine looking redshifts, so I'm inclined to ignore them, but @julienguy<https://urldefense.com/v3/__https:/github.com/julienguy__;!!KGKeukY!0uwV2NDJ1L2hVOhODmmEy0t0vxe1xdthpxig9VhE4Tx9Wtf_9-eV0ar78fUseyOtICsrqvKtani92JWbcf4vCM5GmQ$> expressed some interest in these tiles.

**Comment by araichoor** [comment]
it s formally computed here (it s a per-exposure, per {petal,camera} measurement):
https://github.com/desihub/desispec/blob/019d6fa7d7707d4d3af6a444c927d5c1bf234866/py/desispec/exposure_qa.py#L355-L357
```
            sky_throughput_corr=fitsio.read(sky_filename,"THRPUTCORR")
            petalqa_table[band.upper()+'SKYTHRURMS'][petal]=1.48*np.median(np.abs(sky_throughput_corr-1))
            log.info("petal #{} {} sky throughput rms={:4.3f}".format(petal,band,petalqa_table[band.upper()+"SKYTHRURMS"][petal]))
```

**Comment by schlafly** [comment]
Yeah, that was our question when we saw those alerts too---we don't see them very often.  It's the spread in "TPCORR", which is intended to be a measure of how consistent the sky amplitudes are among the different fibers.  i.e., our expectation is that the sky fibers can be used to subtract the sky from the science fibers, but there is some code that tries to compute a per-fiber sky amplitude (TPCORR) based on the sky lines; this is reporting that the variance in those TPCORR measurements is larger than nominal.

I don't understand it yet.

**Comment by araichoor** [comment]
about those three exposures:
- as reported here https://github.com/desihub/desisurveyops/issues/208, they ve been observed with an unusually small moon separation (35º); but that may not explain it by itself -- I see that for 20230806, we went much closer to the moon (10º<sep<20º!) for some bright tiles, and there was no such warnings
- there were some passing clouds: https://data.desi.lbl.gov/desi/users/raichoor/main-status/spacewatch/spacewatch-20240821.mp4; maybe that, combined with the close-moon, could be the reason? (just guessing!)

**Comment by geordie666** [comment]
For the record, we marked these tiles as `good` in QA. This issue remains open in case anyone wants to dig deeper.

**Comment by geordie666** [comment]
I'm going to close this as "won't fix".
